content/en/storage-providers/advanced-configurations/workers.md

```diff
@@ -0,0 +1,685 @@
+PR2_2K_MIN_MEMORY=2048
+PR2_32G_BASE_MIN_MEMORY=34359738368
+PR2_32G_GPU_UTILIZATION=1
+PR2_32G_MAX_MEMORY=161061273600
+PR2_32G_MAX_PARALLELISM=-1
+PR2_32G_MAX_PARALLELISM_GPU=6
+PR2_32G_MIN_MEMORY=32212254720
+PR2_512M_BASE_MIN_MEMORY=10737418240
+PR2_512M_GPU_UTILIZATION=1
+PR2_512M_MAX_MEMORY=1610612736
+PR2_512M_MAX_PARALLELISM=1
+PR2_512M_MAX_PARALLELISM_GPU=0
+PR2_512M_MIN_MEMORY=1073741824
+PR2_64G_BASE_MIN_MEMORY=68719476736
+PR2_64G_GPU_UTILIZATION=1
+PR2_64G_MAX_MEMORY=204010946560
+PR2_64G_MAX_PARALLELISM=-1
+PR2_64G_MAX_PARALLELISM_GPU=6
+PR2_64G_MIN_MEMORY=64424509440
+PR2_8M_BASE_MIN_MEMORY=8388608
+PR2_8M_GPU_UTILIZATION=1
+PR2_8M_MAX_MEMORY=8388608
+PR2_8M_MAX_PARALLELISM=1
+PR2_8M_MAX_PARALLELISM_GPU=0
+PR2_8M_MIN_MEMORY=8388608
+RU_2K_BASE_MIN_MEMORY=2048
+RU_2K_GPU_UTILIZATION=0
+RU_2K_MAX_MEMORY=2048
+RU_2K_MAX_PARALLELISM=1
+RU_2K_MAX_PARALLELISM_GPU=0
+RU_2K_MIN_MEMORY=2048
+RU_32G_BASE_MIN_MEMORY=1073741824
+RU_32G_GPU_UTILIZATION=0
+RU_32G_MAX_MEMORY=4294967296
+RU_32G_MAX_PARALLELISM=1
+RU_32G_MAX_PARALLELISM_GPU=0
+RU_32G_MIN_MEMORY=4294967296
+RU_512M_BASE_MIN_MEMORY=1073741824
+RU_512M_GPU_UTILIZATION=0
+RU_512M_MAX_MEMORY=1073741824
+RU_512M_MAX_PARALLELISM=1
+RU_512M_MAX_PARALLELISM_GPU=0
+RU_512M_MIN_MEMORY=1073741824
+RU_64G_BASE_MIN_MEMORY=1073741824
+RU_64G_GPU_UTILIZATION=0
+RU_64G_MAX_MEMORY=8589934592
+RU_64G_MAX_PARALLELISM=1
+RU_64G_MAX_PARALLELISM_GPU=0
+RU_64G_MIN_MEMORY=8589934592
+RU_8M_BASE_MIN_MEMORY=8388608
+RU_8M_GPU_UTILIZATION=0
+RU_8M_MAX_MEMORY=8388608
+RU_8M_MAX_PARALLELISM=1
+RU_8M_MAX_PARALLELISM_GPU=0
+RU_8M_MIN_MEMORY=8388608
+UNS_2K_BASE_MIN_MEMORY=2048
+UNS_2K_GPU_UTILIZATION=0
+UNS_2K_MAX_MEMORY=2048
+UNS_2K_MAX_PARALLELISM=1
+UNS_2K_MAX_PARALLELISM_GPU=0
+UNS_2K_MIN_MEMORY=2048
+UNS_32G_BASE_MIN_MEMORY=10485760
+UNS_32G_GPU_UTILIZATION=0
+UNS_32G_MAX_MEMORY=68719476736
+UNS_32G_MAX_PARALLELISM=1
+UNS_32G_MAX_PARALLELISM_GPU=0
+UNS_32G_MIN_MEMORY=60129542144
+UNS_512M_BASE_MIN_MEMORY=1048576
+UNS_512M_GPU_UTILIZATION=0
+UNS_512M_MAX_MEMORY=1073741824
+UNS_512M_MAX_PARALLELISM=1
+UNS_512M_MAX_PARALLELISM_GPU=0
+UNS_512M_MIN_MEMORY=805306368
+UNS_64G_BASE_MIN_MEMORY=10485760
+UNS_64G_GPU_UTILIZATION=0
+UNS_64G_MAX_MEMORY=137438953472
+UNS_64G_MAX_PARALLELISM=1
+UNS_64G_MAX_PARALLELISM_GPU=0
+UNS_64G_MIN_MEMORY=120259084288
+UNS_8M_BASE_MIN_MEMORY=8388608
+UNS_8M_GPU_UTILIZATION=0
+UNS_8M_MAX_MEMORY=8388608
+UNS_8M_MAX_PARALLELISM=1
+UNS_8M_MAX_PARALLELISM_GPU=0
+UNS_8M_MIN_MEMORY=8388608
+```
+{{< /details >}}
+
+## Advanced PoSt worker configurations
+
+{{< alert icon="tip" >}}
+ Use with caution as changing these values to extremes might cause you to miss windowPoSt.
+ {{< /alert >}}
+
+Although the default settings are reasonable you can configure some advanced settings when running the PoSt workers, that can be tested for local optimizations of your hardware.
+
+```
+    --post-parallel-reads value   maximum number of parallel challenge reads (0 = no limit) (default: 128)
+```
```

Comment:
````suggestion
The `--post-parallel-reads` option lets you set an upper boundary of how many challenges it reads from your storage simultaneously. This option is set to 128 by default.

```plaintext
--post-parallel-reads value   maximum number of parallel challenge reads (0 = no limit) (default: 128)
```
````